public/js/test/mochitest/head.js

```diff
@@ -114,21 +130,30 @@ const initDebugger = Task.async(function* (url, ...sources) {
     win: win
   };
 
-  yield Promise.all(sources.map(url => {
-    return waitForState(dbg, state => {
-      return getSources(state).some(s => s.get("url").includes(url));
-    });
-  }));
+  if(sources.length) {
+    info("Waiting on sources: " + sources.join(", "));
+    yield Promise.all(sources.map(url => {
+      function sourceExists(state) {
+        return getSources(state).some(s => s.get("url").includes(url));
```

Comment:
That would never pass. We don't want all the current sources to match this specific url :) We're basically saying "make sure at least one source has this URL".